### PR TITLE
meta: 📇  Updating troubleshooting.md doc to include problem solving w…

### PR DESCRIPTION
**Pull request by frankstallone**
Documenting troubleshooting solutions for users having issues with private Artifactory repositories.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by evocateur** [comment]
ugh, what is happening to CI these days... so many random failures

File: doc/troubleshooting.md [hunk 1]
```diff
@@ -19,6 +19,10 @@ error Command failed with exit code 1.
 If you can upgrade Lerna to said version please do so, or as an alternative 
 solution you can add `--concurrency=1`.
 
+### Artifactory intergration issues
```
Comment (evocateur) [inline doc/troubleshooting.md]: "intergration" -> "integration"

File: doc/troubleshooting.md [hunk 2]
```diff
@@ -19,6 +19,10 @@ error Command failed with exit code 1.
...
+
+If `lerna bootstrap` is failing because you have repositories on a private server make sure you are including that regsitry. Example: `lerna bootstrap -- --regsitry=http://[registry-url]`.
```
Comment (evocateur) [inline doc/troubleshooting.md]: "regsitry" -> "registry"

File: doc/troubleshooting.md [hunk 3]
```diff
@@ -106,3 +110,19 @@ git push --tags --force
 ```
 
 See this [Stackoverflow post](https://stackoverflow.com/questions/5002555/can-a-lightweight-tag-be-converted-to-an-annotated-tag) for more details
+
+### Artifactory intergration issues
```
Comment (evocateur) [inline doc/troubleshooting.md]: This should be titled "Publishing to a private npm registry (Artifactory, npm Enterprise, etc)", as it isn't necessarily specific to Artifactory alone.

**Comment by evocateur** [comment]
You can include "[skip ci]" in your commit messages to avoid running the test suite again (since it's irrelevant to documentation updates)

**Comment by frankstallone** [comment]
Oi, my spelling has gotten worse as I go on. Sorry about that! Thanks for noticing. Updating...

**Pull request merged**